src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -22,27 +22,27 @@ public class SampleDataUtil {
     public static Person[] getSamplePersons() {
```

Comment:
@nigellqx  What do you think about adding a person without initializing the priority field, since priority field will be set to low, and we can test for that

Reply:
I think it is hard because the priority field is needed for the constructor of the person class, however I did the check you propose elsewhere.